lmdeploy/pytorch/engine/engine.py

```diff
@@ -478,7 +478,7 @@ def _batch_stopping_criteria(self, token_ids: torch.Tensor,
 
     @logging_timer('SamplingLogits', logger)
     def async_sampling_logits(self, logits: torch.Tensor,
-                              history_ids: torch.Tensor,
+                              all_ids: torch.Tensor,
```

Comment:
The changes in this file are mainly about renaming "history" to "all". 
What's the motivation?

Reply:
History_ids does not contain prompt_token_ids in the first iteration of genration. This is for both `repetition_panelty` and `custom_logits_processors`.